foundationsync/indico_foundationsync/sync.py

```diff
@@ -125,10 +131,35 @@ def _update_room(self, room, room_data):
         room.is_deleted = False
         room.is_reservable = True
         for k, v in room_data.iteritems():
+            if k == 'manager_group':
+                continue
             if getattr(room, k) != v:
                 setattr(room, k, v)
         db.session.flush()
 
+    def _update_managers(self, room, room_data, room_role_map):
+        new_managers = {room.owner}
+        manager_group = room_data.get('manager_group')
+
+        # add managers from e-group
+        if manager_group is not None:
+            group = GroupProxy(manager_group.strip(), provider='cern-ldap')
```

Comment:
can we also strip `@cern.ch`? right now there's at least one room in the DB that has this (which results in some unnecessary noise in the sentry logs)